modules/container/src/main/java/module-info.java

```diff
@@ -32,5 +32,5 @@
       org.mule.runtime.deployment.model.impl,
       org.mule.runtime.service,
       org.mule.runtime.spring.config,
-      org.mule.test.runner;
+      org.mule.test.runner, org.mule.runtime.deployment.test;
```

Comment:
We shouldn't do this. If you need the "export" in the test, you can use an add-exports in the surefire args